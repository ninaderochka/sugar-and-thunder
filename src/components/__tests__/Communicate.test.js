import React from 'react';
import renderer from 'react-test-renderer';
import { render } from '@testing-library/react';
import Communicate from '../Communicate';

test('Communicate component renders correctly', () => {
  const component = renderer.create(<Communicate />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test('renders the component', () => {
  const { getByText } = render(<Communicate />);
  const heading = getByText(/We can communicate through/i);
  expect(heading).toBeDefined();
});
