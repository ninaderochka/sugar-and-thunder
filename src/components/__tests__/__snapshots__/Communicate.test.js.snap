// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Communicate component renders correctly 1`] = `
<div
  className="w-5/6 h-screen m-auto max-w-full"
>
  <h1
    className="font-poppins font-normal text-5xl ml-3 mb-20 pt-20 uppercase"
  >
    We can communicate through
  </h1>
  <div
    className="grid grid-cols-1 gap-6 max-h-full md:grid-cols-3 lg:grid-cols-3"
  >
    <div
      className="bg-white border-border-grey shadow-shadow-grey drop-shadow-lg font-poppins font-normal box-border p-4 border-2 rounded-[20px] flex content-center m-auto aspect-square w-9/12 md:w-box-width md:h-box-height"
    >
      <div
        className="m-auto flex flex-col space-y-4"
      >
        <img
          alt="voice call"
          className="w-25 m-auto mb-4"
          src="/phone.svg"
        />
        <p
          className="font-poppins font-normal text-5xl text-black text-center mb-4"
        >
          Voice Call
        </p>
        <p
          className="font-poppins font-light text-center text-black/80 w-11/12 m-auto"
        >
          Feeling ready to start a conversation? Give your therapist a voice call and talk your heart out!
        </p>
      </div>
    </div>
    <div
      className="bg-white border-border-grey shadow-shadow-grey drop-shadow-lg font-poppins font-normal box-border p-4 border-2 rounded-[20px] flex content-center m-auto aspect-square w-9/12 md:w-box-width md:h-box-height"
    >
      <div
        className="m-auto flex flex-col space-y-4"
      >
        <img
          alt="chat"
          className="w-25 m-auto mb-4"
          src="/chat.svg"
        />
        <p
          className="font-poppins font-normal text-5xl text-black text-center mb-4"
        >
          Chat
        </p>
        <p
          className="font-poppins font-light text-center text-black/80 w-9/12 m-auto"
        >
          Need to talk to someone? Come have a chat with your therapist!
        </p>
      </div>
    </div>
    <div
      className="bg-white border-border-grey shadow-shadow-grey drop-shadow-lg font-poppins font-normal box-border p-4 border-2 rounded-[20px] flex content-center m-auto aspect-square w-9/12 md:w-box-width md:h-box-height"
    >
      <div
        className="m-auto flex flex-col space-y-4"
      >
        <img
          alt="video"
          className="w-25 m-auto"
          src="/video.svg"
        />
        <p
          className="font-poppins font-normal text-5xl text-black text-center mb-4"
        >
          Video Call
        </p>
        <p
          className="font-poppins font-light text-center text-black/80 w-11/12 m-auto"
        >
          For better experience therapists recommend video calls, but always remember that its a choice!
        </p>
      </div>
    </div>
  </div>
</div>
`;
